rest_lights.cpp

```diff
@@ -1601,6 +1601,10 @@ int DeRestPluginPrivate::setWindowCoveringState(const ApiRequest &req, ApiRespon
             // Legrand invert bri and don't support other value than 0
             targetLiftZigBee = targetLift == 0 ? 100 : 0;
         }
+        else if (taskRef.lightNode->modelId() == QLatin1String("Motor Controller"))
+        {
+            targetLiftZigBee = 100 - targetLift;
+        }

```

Comment:
Please remove the 2 `else if` and move the modelId's to the if-clause with ORs, as you did in `windows_covering.cpp`.